src/main/java/stsjorbsmod/cards/cull/PartingBlow.java

```diff
@@ -0,0 +1,56 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class PartingBlow extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(PartingBlow.class);
+
+    private static final CardRarity RARITY = CardRarity.RARE;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 3;
+    private static final int MAXIMUM_DAMAGE = 100;
+    private static final int UPGRADE_MAXIMUM_DAMAGE = 20;
+
+    public PartingBlow() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        baseDamage = 0;
+        magicNumber = baseMagicNumber = MAXIMUM_DAMAGE;
+    }
+
+    @Override
+    public int calculateBonusBaseDamage() {
+        int maxHP = AbstractDungeon.player.maxHealth;
+        return magicNumber - maxHP;
```

Comment:
Should probably clamp this to a minimum of 0

Reply:
From my tests it appears that it's clamped implicitly somewhere. I'm committing a clamped version for safety.